crates/base/src/rt_worker/implementation/default_handler.rs

```diff
@@ -50,7 +53,23 @@ impl WorkerHandler for Worker {
                         }))
                     }
                 }
-                (Ok(()), _) => Ok(WorkerEvents::EventLoopCompleted(PseudoEvent {})),
+
+                (Ok(()), cpu_usage_ms) => {
+                    if created_rt.is_termination_requested.is_raised() {
+                        static EVENT_RECV_DEADLINE_DUR: Duration = Duration::from_secs(5);
```

Comment:
why do we need to wait for 5 seconds here?

It's nice we can account for CPU time usage on event loop completed events too.

Reply:
@laktek It is an arbitrarily chosen time in case it might not be resolved forever.

Reply:
Note:
I'm revisiting this part, and there's a possibility that if a supervisor unconditionally requests the isolate to terminate, it might not come to this branch, so it might be removed in the future.

Reply:
In that case can we add a comment on it?

Reply:
@laktek 
Yes, I will add that.